app/Nrgi/Services/ElasticSearch/ElasticSearchService.php

```diff
@@ -159,8 +158,12 @@ public function indexMetadata($contract)
             'updated_by'           => json_encode($updated_by),
             'created_at'           => $contract->created_datetime->format('Y-m-d H:i:s'),
             'updated_at'           => $contract->last_updated_datetime->format('Y-m-d H:i:s'),
+            'published_at'         => '',
```

Comment:
empty string in date field? it may cause error if it going to parse via php date

Reply:
Maintained in elasticsearch repo